site/src/App/DocNavigation/DocNavigation.tsx

```diff
@@ -84,52 +84,56 @@ export const DocNavigationItem = ({
         aria-current={active ? 'page' : undefined}
         className={styles.docNavLink}
       >
-        <Box display="flex" alignItems="center" paddingX={navItemPaddingX}>
-          <Box position="relative">
-            <Box
-              display="flex"
-              position="relative"
-              alignItems="center"
-              opacity={active ? undefined : 0}
-              paddingY={navItemPaddingY}
-            >
-              <Box
-                position="absolute"
-                width="full"
-                zIndex={1}
-                bottom={0}
-                className={[
-                  styles.activeUnderline,
-                  styles.activeUnderlineColor[lightness.lightMode],
-                  styles.activeUnderlineColor[lightness.darkMode],
-                ]}
-              />
-              <Text size="standard" weight="strong">
-                {children}
-              </Text>
-              {badgeElement}
-            </Box>
+        <Box
+          position="relative"
+          display="flex"
+          alignItems="center"
+          paddingX={navItemPaddingX}
+        >
+          {/* Active, strong title */}
+          <Box
```

Comment:
While we are here let's improve the markup. We should really be using `span` elements inside of `a` elements.
```suggestion
          <Box
            component="span"
```